core/spec/models/spree/refund_spec.rb

```diff
@@ -92,6 +92,11 @@
         subject
       end
 
+      it 'should update the payment total' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.